localstack/services/opensearch/cluster.py

```diff
@@ -248,25 +249,22 @@ def __init__(self, enabled: bool, endpoint: str) -> None:
             self.url = None
 
 
-class EdgeProxiedOpensearchCluster(Server):
+class EdgeProxiedOpensearchServer(RegisteredProxyServer):
     """
-    Opensearch-backed Server that can be routed through the edge proxy using an UrlMatchingForwarder to forward
-    requests to the backend cluster.
+    Opensearch-backed Server that can be routed through the edge proxy making use of the rule adding specified in
+    RegisteredProxyServer. With these rules, requests are forwarded to the backend cluster.
     """
 
     def __init__(self, url: str, arn: str, version=None) -> None:
         self._url = urlparse(url)
 
-        super().__init__(
-            host=self._url.hostname,
-            port=self._url.port,
-        )
+        super().__init__(url)
         self._version = version or self.default_version
         self.arn = arn
 
         self.cluster = None
         self.cluster_port = None
-        self.proxy = None
+        self.route_rules = []
```

Comment:
As far as I can see, this list is empty and stays empty?

Reply:
yes, this was pulled into the parent class, thanks!